goalie/mesh_seq.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, time_partition, initial_meshes, **kwargs):
         :type parameters: :class:`~.AdaptParameters`
         """
         self.time_partition = time_partition
-        self.fields = time_partition.fields
+        self.fields = {field: None for field in time_partition.fields}
```

Comment:
There was an unresolved comment related to this line: https://github.com/mesh-adaptation/goalie/pull/182#discussion_r1580610436

Reply:
You're right, thanks! Done in f93f7241c0d456b1e68940ce20ac821932d1820a